src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -269,7 +310,16 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
             bannerPolicy,
         }: Props = this.props;
 
-        const { accessStats, accessStatsError, file, fileError, isLoading }: State = this.state;
+        const {
+            accessStats,
+            accessStatsError,
+            classificationInfo,
+            classificationError,
+            file,
+            fileError,
+            isLoadingAccessStats,
```

Comment:
Do we need 2 loading states? Are they being used locally? If not, then you can just:

componentDidMount() {
    this.fetchData();
}

fetchData() {
     if (hasAccessStats || hasClassification) {
         this.setState({ isLoading: true });
     }
}

success/error callbacks for each of the 2, setState({ isLoading: false })

Reply:
I do notice you are &&ing below, shouldn't the details properties show when avail...like not wait for everything to load?

Reply:
If classification comes back before access stats does, then we have no loader, but ALSO no access stats until they're loaded (which looks weird). I agree they should show before everything is ready, but the UI looks funky when things are popping in *without* a loading spinner